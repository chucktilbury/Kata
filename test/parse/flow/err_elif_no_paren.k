;-------------------------------------------------------------------------------
; error elif imbalanced parens
;-------------------------------------------------------------------------------

func name()() {

    if(comp.name == 0) {
        bulart = 0
    }
    elif(1+2 {
        goop()()
    }
}