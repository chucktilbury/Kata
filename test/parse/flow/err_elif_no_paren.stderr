Syntax: /home/chuck/Src/Kata/test/parse/flow/err_elif_no_paren.k: 10: 15: expected a ')' but got {
Syntax: /home/chuck/Src/Kata/test/parse/flow/err_elif_no_paren.k: 11: 15: expected function body element but got )
Syntax: /home/chuck/Src/Kata/test/parse/flow/err_elif_no_paren.k: 11: 17: expected function body but got )
Syntax: /home/chuck/Src/Kata/test/parse/flow/err_elif_no_paren.k: 13: 2: expected module item or end of file but got }
Warning: /home/chuck/Src/Kata/test/parse/flow/err_elif_no_paren.k: 13: 2: module is empty
